.github/workflows/linting-and-tests.yml

```diff
@@ -243,7 +243,7 @@ jobs:
       matrix:
         grafana_version:
           - 10.3.0
-          - latest
```

Comment:
let's add `latest` back, seems it's the team's preference for now https://raintank-corp.slack.com/archives/C06K1MQ07GS/p1724853109521819